From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Sunu Aziz Rahayu <sunuazizrahayu@users.noreply.github.com>
Date: Fri, 26 Sep 2026 00:00:00 +0700
Subject: [PATCH] base: remove non-existent healthd and vr_hwc modules

Kedua modul ini sudah tidak ada di Lineage 23.2 / Android 16,
sehingga kati gagal:

  lineage_arm64_bvN4.mk includes non-existent modules
  in PRODUCT_PACKAGES: healthd, vr_hwc

---
 base.mk | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/base.mk b/base.mk
index 0000000..0000000 100644
--- a/base.mk
+++ b/base.mk
@@ -199,9 +199,7 @@ PRODUCT_PACKAGES += \
     android.hardware.biometrics.fingerprint@2.1-service.oplus.compat \

 PRODUCT_PACKAGES += \
-    vr_hwc \
-    curl \
-    healthd \
+    curl

 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
     debug.fdsan=warn_once \
--
2.54.0
